### PR TITLE
Add LinwoodDev.Flow.Nightly

### PR DESCRIPTION
**Pull request by CodeDoctorDE**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103698)

### CODE REVIEW

File: manifests/l/LinwoodCloud/Flow/Nightly/0.1.0/LinwoodCloud.Flow.Nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: LinwoodCloud.Flow.Nightly
+PackageVersion: 1.0.0
```
Comment (russellbanks) [inline manifests/l/LinwoodCloud/Flow/Nightly/0.1.0/LinwoodCloud.Flow.Nightly.yaml:5]:
```suggestion
PackageVersion: 0.1.0
```

File: manifests/l/LinwoodCloud/Flow/Nightly/0.1.0/LinwoodCloud.Flow.Nightly.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
```
Comment (russellbanks) [inline manifests/l/LinwoodCloud/Flow/Nightly/0.1.0/LinwoodCloud.Flow.Nightly.yaml:2]:
```suggestion
# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
```

File: manifests/l/LinwoodCloud/Flow/Nightly/0.1.0/LinwoodCloud.Flow.Nightly.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+PackageIdentifier: LinwoodCloud.Flow.Nightly
+PackageVersion: 1.0.0
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.1.0
```
Comment (russellbanks) [inline manifests/l/LinwoodCloud/Flow/Nightly/0.1.0/LinwoodCloud.Flow.Nightly.yaml:8]:
```suggestion
ManifestVersion: 1.2.0
```

**Review by russellbanks — CHANGES_REQUESTED** [review]
The version file doesn't match the rest

**Review by stephengillie — APPROVED** [review]

**Pull request merged**